spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -83,3 +83,13 @@ See xref:kafka/seek.adoc#seek[Seek API Docs] for more details.
 For Spring Boot applications which define an application name, this name is now used
 as a default prefix for auto-generated client IDs for certain client types.
 See xref:kafka/connecting.adoc#default-client-id-prefixes[Default client ID prefixes] for more details.
+
+[[get-listener-containers-matching]]
+== Retrieve MessageListenerContainers with by matching a Predicate
+
+Starting with version 3.2, the `ListenerContainerRegistry` introduces the `getListenerContainersMatching` method.
+This method filters `MessageListenerContainer` instances based on a `Predicate<String>` applied to their IDs.
```

Comment:
After your great docs I wonder if we could extend this new feature to something like this `BiPredicate<String, MessageListenerContainer>`. This way we may filter desired containers not only by their ids, but via some other option form the container itself.

WDYT?

Reply:
Makes sense 👍🏼
I wonder if we should have both or just `BiPredicate<String, MessageListenerContainer>` 🤔
Extended feature sounds very powerful, but might be too verbose in someway, knowing that `id` is the key used to access the `MessageContainerListener`.

Reply:
Since the `String, MessageListenerContainer` pair in `BiPredicate` technically represents the key-value pair stored in the registry, how about having both?

Reply:
Should we do a separate PR or do everything in this one?

Reply:
This one is OK.
Thanks

Reply:
Alright, hopefully it would probably take an hour or two